libs/components/ag-grid/src/lib/styles/_modern-light.scss

```diff
@@ -3,6 +3,10 @@
 
 .ag-theme-sky-modern-light {
   ag-grid-angular:not(.sky-ag-grid-editable) {
+    .ag-row-hover::before {
+      background-color: transparent;
+    }
+
```

Comment:
future proofing light mode jic angular adds a background color for row hover in light mode in the future